packages/dataviews/src/components/dataviews-selection-checkbox/index.tsx

```diff
@@ -29,20 +28,10 @@ export default function DataViewsSelectionCheckbox< Item >( {
 }: DataViewsSelectionCheckboxProps< Item > ) {
 	const id = getItemId( item );
 	const checked = ! disabled && selection.includes( id );
-	let selectionLabel;
-	if ( titleField?.getValue && item ) {
-		// eslint-disable-next-line @wordpress/valid-sprintf
-		selectionLabel = sprintf(
-			checked
-				? /* translators: %s: item title. */ __( 'Deselect item: %s' )
-				: /* translators: %s: item title. */ __( 'Select item: %s' ),
-			titleField.getValue( { item } )
-		);
-	} else {
-		selectionLabel = checked
-			? __( 'Select a new item' )
-			: __( 'Deselect item' );
-	}
+
+	// Fallback label to ensure accessibility
+	const selectionLabel = titleField?.getValue?.( { item } ) || 'Select item';
```

Comment:
I'm thinking about making the fallback title "(no title)". I feel like that would improve the consistency of what's read out when focused.

For example, with NVDA you can expect:

- When the pattern itself is focused: `(no title)  button`
- When the pattern's checkbox is focused: `(no title)  check box  not checked`

What do you think?

Reply:
Using "(no title)" aligns well with the standard behaviour of accessibility tools and avoids uncertainity compared to a more generic fallback like "Select item." This makes it clear to users that the item doesn't have a title. So can I update the condition like this ?

`const selectionLabel = titleField?.getValue?.( { item } ) || '(no title)';`

Reply:
Yes, but you also need a translation function.

```javascript
const selectionLabel = titleField?.getValue?.( { item } ) || __( '(no title)' );
```

Reply:
@t-hamano - Updated the mentioned changes.